gradle.properties

```diff
@@ -1,5 +1,5 @@
 kotlin.code.style=official
 kotlin_version=1.5.0
-release_version=0.0.2
+release_version=0.0.3
```

Comment:
Please update the minor version because the minor version of the common lib has been updated